crates/handler/src/validation.rs

```diff
@@ -74,8 +74,10 @@ where
         context: &Self::Context,
     ) -> Result<InitialAndFloorGas, Self::Error> {
         let spec = context.cfg().spec().into();
-        validate_initial_tx_gas::<&Self::Context, InvalidTransaction>(context, spec)
-            .map_err(Into::into)
+        let initial_and_floor_gas =
+            validate_initial_tx_gas(context.tx(), spec).map_err(InvalidTransaction::from)?;
+
+        Ok(initial_and_floor_gas)
```

Comment:
```suggestion
        validate_initial_tx_gas(context.tx(), spec).map_err(From::from)
```

should work

Reply:
Addressed in https://github.com/bluealloy/revm/pull/2006/commits/fee24bc012334a633e73f6fa0574eb0dd81c2445